content/TrueCommand/SystemManagement/_index.md

```diff
@@ -50,18 +49,20 @@ Clicking *APPS* (Scale) or *Jails* (TrueNAS 12.x) opens a window that allows use
 The *Options* menu has several shortcuts to simple tasks.
```

Comment:
For consistency all on-screen elements are bold, Suggest making Options bold not italics